src/types/identity.ts

```diff
@@ -0,0 +1,50 @@
+import { body, ValidationChain } from "express-validator";
+import { nameof } from "./util/nameof";
+import { VALID_LOC_VALUES } from "../constants";
+
+export default interface Identity {
+  vot?: string;
+  vtm?: string;
+  iss?: string;
+  sub?: string;
+  nbf?: number;
+  exp?: number;
+  aud?: string;
+  iat?: number;
+  jti?: string;
+}
+
+export const generateIdentityPropertyValidators = (
+  parent?: string
+): ValidationChain[] => {
+  const prefix = parent ? `${parent}.` : "";
+  return [
+    body(`${prefix}.${nameof<Identity>("vot")}`)
+      .isIn(VALID_LOC_VALUES)
+      .optional({ values: "null" }),
+    body(`${prefix}${nameof<Identity>("vtm")}`)
+      .isURL()
+      .optional({ values: "null" }),
+    body(`${prefix}${nameof<Identity>("iss")}`)
+      .isURL()
+      .optional({ values: "null" }),
+    body(`${prefix}${nameof<Identity>("sub")}`)
+      .isURL()
```

Comment:
Quite a few of have the `isURL` method called, I wouldn't expect most of them to be valid URLs. Is this correct?

Reply:
I think you're right - _iss_ and _sub_ should just validated to be strings.

Reply:
Fixed

Reply:
Thanks !